tests/test_messages.py

```diff
@@ -4,7 +4,8 @@
 from eth_utils import is_same_address
 
 from raiden_libs.utils import sign_data, private_key_to_address, encode_hex
-from raiden_libs.messages import BalanceProof, Message, FeeInfo, MonitorRequest
+from raiden_libs.messages import (BalanceProof, Message, FeeInfo, MonitorRequest, PathsRequest,
```

Comment:
code style.

Reply:
Following https://www.python.org/dev/peps/pep-0328/, as long imports are not covered by our style-guide.

Reply:
Sry, this is what we use: https://github.com/raiden-network/raiden/blob/master/raiden/app.py#L8

Reply:
Done!